README.md

```diff
@@ -17,13 +17,14 @@ A `Node.js` based command-line client for [tldr](https://github.com/tldr-pages/t
 npm install -g tldr
 ```
 
-If you have trouble with the post-install script, try the following commands:
+If you are installing with sudo, pass the `--user` parameter:
 
 ```bash
-npm install -g --ignore-scripts tldr
-tldr --update
+sudo npm install -g tldr --user=<logged_in_user>
```

Comment:
I don't think we should be aiming for support in `cmd.exe` -- does the rest of the tool even work there (e.g. the color-coded output)?